preprocessing/preproccessing_one_hot_encoding.py

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
+from pandas import DataFrame
+from typing import Tuple
+
+from sklearn.preprocessing import OneHotEncoder
+
+from preprocessing import get_preprocessed_brfss_dataset, diabetes_columns, oversample_dataset, \
+    undersample_dataset
```

Comment:
PyCharm underlines this import red, but it is the only way I could make it work when I actually run the code. Any ideas @Niklas-23 ?

Reply:
I think this must be a problem with your local setup, I don't understand how you can run the code when PyCharm underlines this as wrong for you too. I added a unit test showing the error.